static/js/tabbed-content.js

```diff
@@ -213,6 +215,15 @@
         tabContainer.querySelectorAll("[aria-controls]")
       );
 
+      // remove any button that invokes a modal
+      // for example, there might be a "get in touch" button within a tab
+      const modalTriggerIndex = tabs.findIndex((tab) =>
+        tab.classList.contains("js-invoke-modal")
+      );
+      if (modalTriggerIndex !== -1) {
+        tabs.splice(modalTriggerIndex, 1);
+      }
```

Comment:
This would only remove the first element with "js-invoke-modal" class what if there are multiple instances of it?
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex